src/components/AddApplication/AddApplication.js

```diff
@@ -113,7 +116,16 @@ const AddApplication = () => {
         return  (
             <WizardBody
                 goToSources={goToSources}
-                step={<LoadingStep />}
+                step={<LoadingStep
+                    customText={intl.formatMessage({
```

Comment:
Why is there the `intl.formatMessage` instead of a formatted message?

Reply:
proptypes.string

I will fix it in the sources package, but after the V2 is fixed

Reply:
Ah i though so.